index.html

```diff
@@ -7277,8 +7277,78 @@ <h2><code>application/td+json</code> Media Type Registration</h2>
       <dt>Change controller:</dt>
       <dd><abbr title="World Wide Web Consortium">W3C</abbr></dd>
     </dl>
-
     </section>
+
+    <section id="media-type-section-tm">
+        <h2><code>application/tm+json</code> Media Type Registration</h2>
+        <dl>
+          <dt>Type name:</dt>
+          <dd>application</dd>
+          <dt>Subtype name:</dt>
+          <dd>tm+json</dd>
+          <dt>Required parameters:</dt>
+          <dd>None</dd>
+          <dt>Optional parameters:</dt>
+          <dd>None</dd>
+          <dt>Encoding considerations:</dt>
+          <dd>See <a href="https://tools.ietf.org/html/rfc6839#section-3.1">RFC&nbsp;6839, section 3.1</a>.
+          </dd>
+          <dt>Security considerations:</dt>
+          <dd>See 
+          <a href="#sec-security-consideration-context-tampering"></a>,
+          <a href="#sec-security-consideration-execution"></a>, 
+          <a href="#sec-security-consideration-jsonld-expansion"></a>,
+          and
+          <a href="#sec-privacy-consideration-context"></a> in
+          the published specification.
+          </dd>
+          <dt>Interoperability considerations:</dt>
+          <dd>See <a href="https://tools.ietf.org/html/rfc8259">RFC&nbsp;8259</a>.
+            <p>
+            Rules for processing both conforming and non-conforming content are
+            defined in this specification.
+            </p>
+          </dd>
+          <dt>Published specification:</dt>
+          <dd>
+            <a href="https://www.w3.org/TR/wot-thing-description11/">W3C 
+            Web of Things (WoT) Thing Description 1.1</a>
+         </dd>
+          <dt>Applications that use this media type:</dt>
+          <dd>
+            All participating entities in the W3C Web of Things, that is,
+            <a>Things</a>, <a>Consumers</a>, and Intermediaries as defined in the
+            <a href="https://www.w3.org/TR/wot-architecture11/">W3C 
+        Web of Things (WoT) Architecture 1.1</a> document.
+          </dd>
+          <dt>Fragment identifier considerations:</dt>
+          <dd>See <a href="https://tools.ietf.org/html/rfc6839#section-3.1">RFC&nbsp;6839, section 3.1</a>.
```

Comment:
JSON Pointer is [RFC6901](https://w3c.github.io/wot-thing-description/#bib-rfc6901)].
RFC 6839 is just general guidance on how tags work but says that "if there is an extension the fragment identifier can be defined" but the parent is otherwise inherited, but JSON itself (the parent) does not define a fragment identifier.

I think we should *explicitly* state that JSON Pointer (the RFC above) is the default fragment identifier for TMs (AND TDs...)

Reply:
done